src/farm.ts

```diff
@@ -0,0 +1,27 @@
+import fs from 'node:fs'
```

Comment:
Since the farm does not load other types of files by default, it is necessary to wrap it with a plugin in advance to ensure normal operation without affecting other frameworks.